src/components/Setup/Setup.tsx

```
@@ -0,0 +1,12 @@
+'use client'
+import { createElement, useEffect, useState } from 'react'
+import * as _consts from './Setup.consts'
+
+export const Setup = ({}) => {
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)